docs/conf.py

```diff
@@ -51,7 +58,7 @@
 master_doc = "index"
 
 # General information about the project.
-project = u"FV3GFS"
+project = u"fv3gfs-wrapper"
 copyright = u"2019, Vulcan Technologies, LLC"
```

Comment:
Does the copyright year need to be updated?

Reply:
Good point.